Python/Product/Ipc.Json/Connection.cs

```diff
@@ -253,6 +270,8 @@ public sealed class Connection : IDisposable {
                     var type = packet["type"].ToObject<string>();
                     var seq = packet["seq"].ToObject<int?>();
 
+                    Debug.WriteLine(string.Format("Received packet seq={0}", seq));
```

Comment:
I'll remove it. The output printed by ProcessResponse should be sufficient. I added it when I was being paranoid when I was suffering from the TCS issue where my process messages thread was off executing client request continuation code instead of processing messages.

---
In reply to: [107988499](https://github.com/Microsoft/PTVS/pull/2323#discussion_r107988499) [](ancestors = 107988499)